examples/c-bsp-test/RISC-V-Toolchain.cmake

```diff
@@ -0,0 +1,7 @@
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR riscv64)
+
+set(CMAKE_C_COMPILER /opt/homebrew/Cellar/riscv-gnu-toolchain/main/bin/riscv64-unknown-elf-gcc)
+set(CMAKE_ASM_COMPILER /opt/homebrew/Cellar/riscv-gnu-toolchain/main/bin/riscv64-unknown-elf-as)
+set(CMAKE_LINLKER /opt/homebrew/Cellar/riscv-gnu-toolchain/main/bin/riscv64-unknown-elf-ld)
```

Comment:
Can we read these from path using conventional environment variables, e.g. "CC"? Alternatively, CMake also allows making variables "user-defined". The user should be prompted for these upon first running cmake.

Reply:
CC is now used. Toolchain file deleted.